app/models/representative.rb

```diff
@@ -13,12 +13,20 @@ def self.contains_official(official)
     false
   end
 
+  def self.build_or_create_rep(official, ocdid_temp, title_temp)
+    if contains_official(official)
+      Representative.new({ name: official.name, ocdid: ocdid_temp,
+        title: title_temp })
+    else
+      Representative.create!({ name: official.name, ocdid: ocdid_temp,
```

Comment:
```suggestion
      Representative.create({ name: official.name, ocdid: ocdid_temp,
```
It's still `create!` haha.

Reply:
Resolved by [baead5a](https://github.com/cs169/fa23-chips-10.5-12/pull/20/commits/baead5ab4cc9c2aef0947a008a83c6b920600fc3)